features/org.wso2.developerstudio.eclipse.bps.feature/feature.xml

```diff
@@ -34,6 +34,42 @@
       %license
    </license>
 
+
+	 <requires>
```

Comment:
Fixed in https://github.com/wso2/devstudio-tooling-bps/pull/8/commits/f3e01a45dfa01411dffa789ae02463b98068ee3c